target/linux/ath79/image/nand.mk

```diff
@@ -1,3 +1,16 @@
+define Build/dw-headers
+	head -c 4 $@ >> $@.tmp && \
+	printf '\x00\x00\x00\x00\x00\x02\x04\x00' >> $@.tmp && \
```

Comment:
The header code has changed a lot from manatail's original script, but still this line remains almost the same.
If I were you, I would give credit to the original author in the commit message...

Personally I think `head -c 8 /dev/zero >> ...` would suffice here, though.

Reply:
It's not a magic number. It's an internal version code. It will be evident if you compare the value from your nand image dump with the version info from the router setting page. I guess any value will fit there.

Reply:
I followed the method you suggested and it really worked. Your assumption was really right. You're such an expert!
I'll update the header generation function as you provided.